docs-site/website/versioned_docs/version-2.0.0/onnx.md

```diff
@@ -1,7 +1,6 @@
 ---
-id: version-2.0.0-onnx
```

Comment:
@joddiy hi for versioned_docs you have to keep the predix

Reply:
Hi, @FeynmanDNA , sorry for late, just fix it.